src/geo.rs

```diff
@@ -64,8 +67,8 @@ pub fn subdivide_contour(contour: &[Coordinate]) -> Vec<Coordinate> {
     /// Returns a list of coordinates making up the line between c0 and c1 (excluding c1 itself)
     /// but split into small enough chunks.
     fn split_line(c0: Coordinate, c1: Coordinate) -> Vec<Coordinate> {
-        let largest_angle = largest_angle(c0, c1);
-        if largest_angle > MAX_COORDINATE_ANGLE_RAD_CONTOURS {
+        let line_error = error(c0, c1);
```

Comment:
Just a thought:
Since we are working with a sphere, and the error is fixed there should be a chord of length N where the length will have a result above the error, if we calculate the length of that chord. We should be able to know if it is within or outside the margin of error just based on its length?

Reply:
This is a good simplification. I'd argue that the readability is more important than the performance here. But it was possible to implement that with at least as readable code. So I went ahead and made this change. Thanks